plugins/modules/postgresql_ext.py

```diff
@@ -272,60 +291,71 @@ def ext_get_versions(cursor, ext):
       ext (str) -- extension name
     """
 
+    current_version = None
+    params = {}
+    params['ext'] = ext
+
     # 1. Get the current extension version:
     query = ("SELECT extversion FROM pg_catalog.pg_extension "
              "WHERE extname = %(ext)s")
 
-    current_version = None
-    cursor.execute(query, {'ext': ext})
+    cursor.execute(query, params)
+
     res = cursor.fetchone()
     if res:
         current_version = res[0]
 
     # 2. Get available versions:
     query = ("SELECT version FROM pg_available_extension_versions "
              "WHERE name = %(ext)s")
-    cursor.execute(query, {'ext': ext})
-    res = cursor.fetchall()
 
-    available_versions = parse_ext_versions(current_version, res)
+    cursor.execute(query, params)
+
+    available_versions = [r[0] for r in cursor.fetchall()]
```

Comment:
I'll readily admit I don't now the intricacies of lists vs sets in python, but I saw you had previously used a list for this variable, so stuck with that to avoid needing possible further code changes elsewhere. Found that quick one-liner for turning a single-value tuple into a list so thought that was a nice simplification. Would you prefer this instead?
```
available_versions = []
for r in cursor.fetchall():
  available_versions.append([r0])
```

Reply:
i wrote this several years ago:)
i meant something like `available_versions = set(r[0] for r in cursor.fetchall())`. Anyway, it doesn't change much, so if you prefer the list, feel free to mark this suggesting as resolved.

Reply:
Changed to the given code example and seems to work without any issues with just that change.